microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/activation/ActivationPropertiesUnsetTest.java

```diff
@@ -0,0 +1,80 @@
+package org.jboss.eap.qe.microprofile.jwt.activation;
+
+import org.jboss.arquillian.container.test.api.Deployer;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.eap.qe.microprofile.jwt.testapp.jaxrs.NonJwtJaxRsTestApplication;
+import org.jboss.eap.qe.microprofile.jwt.testapp.jaxrs.SecuredJaxRsEndpoint;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Set of negative tests for subsystem activation.
+ */
+@RunWith(Arquillian.class)
+public class ActivationPropertiesUnsetTest {
+
+    private static final String NON_JWT_AUTH_ACTIVATION_PROPERTY_DEPLOYMENT_NAME = "PROPERTIES_UNSET_DEPLOYMENT";
+    private static final String UNSET_AUTH_ACTIVATION_PROPERTY_DEPLOYMENT_NAME = "OTHER_VALUE_DEPLOYMENT";
```

Comment:
Sorry if I am missing something but it seems to me that during the first round of review these two constants were switched.
What I mean: 

* `UNSET_AUTH_ACTIVATION_PROPERTY_DEPLOYMENT_NAME` - i.e. no authentication is activated because all related properties are unset  => "PROPERTIES_UNSET_DEPLOYMENT"

* `NON_JWT_AUTH_ACTIVATION_PROPERTY_DEPLOYMENT_NAME` - i.e. authentication is activated through some different (non JWT) mechanism => "OTHER_VALUE_DEPLOYMENT"

If the names I suggested shouldn't look clear to you, please feel free to choose something else, e.g. I see you used very clear names in other tests, as for instance `DEPLOYMENT_WITH_VALID_KEY`.

Reply:
Done, thanks.